content/en/Getting started/glossary.md

```diff
@@ -73,7 +73,7 @@ relates to pentests of web application technical security controls.
 
 ## Attacker
 
-Sometimes also known as a Threat Actor, Malicious Hacker, "Black-hat Hacker," or "Cracker." May be 
+Sometimes also known as a threat actor, malicious hacker, "black hat hacker," or "cracker." May be 
```

Comment:
@mjang-cobalt I'm working on the Word List for the UI Text Style Guide and decided to submit this minor PR.

The spelling of _black hat hacker_ is inconsistent across different sources. I relied on https://www.merriam-webster.com/dictionary/black%20hat

Reply:
Makes sense. X-verified with https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/security-terms.